common/src/main/resources/db/changelog/test_only/truncate-fake-data.sql

```diff
@@ -0,0 +1,9 @@
+--liquibase formatted sql
```

Comment:
When/how does this get called?

Reply:
Upon setup of the Postgres SQL Container.